Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -340,35 +311,99 @@ script:
       description: The number of times the vulnerability was found.
       type: number
     - contextPath: TenableIO.Vulnerabilities.VulnerabilityState
-      description: The current state of the reported vulnerability (Active, Fixed, New, etc.)
-      type: string
-  - arguments:
-    - default: true
-      description: The unique ID of the Scan.
-      isArray: false
-      name: scanId
+      description: The current state of the reported vulnerability (Active, Fixed,
+        New, etc.)
+      type: string
+    description: Get a list of up to 5000 of the vulnerabilities recorded for a given
+      asset.
+  - name: tenable-io-get-scan-status
+    arguments:
+    - name: scanId
       required: true
-      secret: false
-    deprecated: false
-    description: |-
-      Check the status of a specific scan using its ID. The status can
-      hold following possible values : Running, Completed and Empty (Ready to run).
-    execution: false
-    name: tenable-io-get-scan-status
+      default: true
+      description: The unique ID of the Scan.
     outputs:
     - contextPath: TenableIO.Scan.Id
       description: The unique ID of the scan specified.
       type: string
     - contextPath: TenableIO.Scan.Status
       description: The status of the scan specified.
       type: string
-  isfetch: false
-  longRunning: false
-  longRunningPort: false
+    description: |-
+      Check the status of a specific scan using its ID. The status can
+      hold following possible values : Running, Completed and Empty (Ready to run).
+  - name: tenable-io-pause-scan
+    arguments:
+    - name: scanId
+      required: true
+      description: The ID of the scan
+    outputs:
+    - contextPath: TenableIO.Scan.Id
+      description: The unique id of the scan.
+    - contextPath: TenableIO.Scan.Status
+      description: The status of the scan (completed, aborted, imported, pending,
+        running, resuming, canceling, cancelled, pausing, paused, stopping, stopped).
+    description: Pauses a running scan given the scan ID
+  - name: tenable-io-resume-scan
+    arguments:
+    - name: scanId
+      required: true
+      description: Scan ID to be resumed
+    outputs:
+    - contextPath: TenableIO.Scan.Id
+      description: The unique id of the scan.
+    - contextPath: TenableIO.Scan.Status
+      description: The status of the scan (completed, aborted, imported, pending,
+        running, resuming, canceling, cancelled, pausing, paused, stopping, stopped).
+    description: Resumes a paused scan given the scan ID
+  - name: tenable-io-add-tags
+    arguments:
+    - name: payload
+      required: true
+      description: Payload for REST call
+      isArray: true
+    description: Add tags to Tenable tenant
+  - name: tenable-io-resume-scans
+    arguments:
+    - name: scanIds
+      required: true
+      description: Comma separated scan ID's
+      isArray: true
+    description: Resume all scans inputted as an array
```

Comment:
Please remove as we are going to use `tenable-io-resume-scan` command.

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed command from code